Client/Frontend/UserScript/UserScript.swift

```diff
@@ -0,0 +1,16 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import WebKit
+
+func getUserScript() -> WKUserScript? {
+    if let path = NSBundle.mainBundle().pathForResource("UserScript", ofType: "js") {
```

Comment:
This should be the basis of a `NoImageModeHelper` tab helper.

It can live in `Frontend/Browser`.